### PR TITLE
Privacy set up

### PR DESCRIPTION
**Pull request by MadelineMurray**
Privacy set up

### CODE REVIEW

File: docs/Privacy/Configuring-Privacy.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+pantheon --data-path=data --genesis-file=..\ibftGenesis.json --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT,EEA --host-whitelist=* --rpc-http-cors-origins="all" --privacy-enabled --privacy-url=http://127.0.0.1:8888 --privacy-public-key-file=Orion\nodeKey.pub   
+```
+
+The command line specifies privacy options: 
+
+* [`--privacy-enabled`](../Reference/Pantheon-CLI-Syntax.md#privacy-enabled) enables privacy
+* [`--privacy-url`](../Reference/Pantheon-CLI-Syntax.md#privacy-url) specifies the Orion node URL (`clienturl` in `orion.conf`)
+* [`--privacy-public-key-file`](../Reference/Pantheon-CLI-Syntax.md#privacy-public-key-file) specifies the file containing
+Orion node public key (created in [3. Generate Orion Keys](#3-generate-orion-keys))
+* [`--rpc-http-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-api) includes `EEA` enabling privacy JSON-RPC API methods.
```
Comment (antonydenyer) [inline docs/Privacy/Configuring-Privacy.md]: To run with the default RPC modules along with the eea privacy module you need to specify them explicitly eg `--rpc-http-api=ETH,NET,WEB3,EEA`

File: docs/Privacy/Configuring-Privacy.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+Copy the enode URL to specify Node-1 as the bootnode in the following steps. 
+
+![Node 1 Enode URL](../images/EnodeStartup.png)
+
+### 7. Start Pantheon Node-2 
+
+In the `Node-2` directory, start Pantheon Node-2 specifying the Node-1 enode URL copied when starting Node-1 as the bootnode:
+ 
+```bash tab="MacOS"
+pantheon --data-path=data --genesis-file=../ibftGenesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8546 --privacy-enabled --privacy-url=http://127.0.0.1:8889 --privacy-public-key-file=Orion/nodeKey.pub
```
Comment (antonydenyer) [inline docs/Privacy/Configuring-Privacy.md]: --rpc-http-api=ETH,NET,WEB3,IBFT,EEA

File: docs/Privacy/Configuring-Privacy.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+### 7. Start Pantheon Node-2 
+
+In the `Node-2` directory, start Pantheon Node-2 specifying the Node-1 enode URL copied when starting Node-1 as the bootnode:
+ 
+```bash tab="MacOS"
+pantheon --data-path=data --genesis-file=../ibftGenesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8546 --privacy-enabled --privacy-url=http://127.0.0.1:8889 --privacy-public-key-file=Orion/nodeKey.pub    
+```
+
+```bash tab="Windows"
+pantheon --data-path=data --genesis-file=..\ibftGenesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8546 --privacy-enabled --privacy-url=http://127.0.0.1:8889 --privacy-public-key-file=Orion\nodeKey.pub
```
Comment (NicolasMassart) [inline docs/Privacy/Configuring-Privacy.md]: same here `--rpc-http-api=ETH,NET,WEB3,IBFT,EEA`

File: docs/Privacy/Configuring-Privacy.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+!!!note
+    The [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) and [`--rpc-http-port`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-port)
+    options are not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#exposing-ports).
+
+### 8. Start Pantheon Node-3
+
+In the `Node-3` directory and start Pantheon Node-3 specifying the Node-1 enode URL copied when starting Node-1 as the bootnode: 
+
+```bash tab="MacOS"
+pantheon --data-path=data --genesis-file=../ibftGenesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30305 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8547 --privacy-enabled --privacy-url=http://127.0.0.1:8890 --privacy-public-key-file=Orion/nodeKey.pub
```
Comment (NicolasMassart) [inline docs/Privacy/Configuring-Privacy.md]: --rpc-http-api=ETH,NET,WEB3,IBFT,EEA

File: docs/Privacy/Configuring-Privacy.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+### 8. Start Pantheon Node-3
+
+In the `Node-3` directory and start Pantheon Node-3 specifying the Node-1 enode URL copied when starting Node-1 as the bootnode: 
+
+```bash tab="MacOS"
+pantheon --data-path=data --genesis-file=../ibftGenesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30305 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8547 --privacy-enabled --privacy-url=http://127.0.0.1:8890 --privacy-public-key-file=Orion/nodeKey.pub   
+```
+
+```bash tab="Windows"
+pantheon --data-path=data --genesis-file=..\ibftGenesis.json --bootnodes=<Node-1 Enode URL> --p2p-port=30305 --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT --host-whitelist=* --rpc-http-cors-origins="all" --rpc-http-port=8547 --privacy-enabled --privacy-url=http://127.0.0.1:8890 --privacy-public-key-file=Orion\nodeKey.pub
```
Comment (NicolasMassart) [inline docs/Privacy/Configuring-Privacy.md]: --rpc-http-api=ETH,NET,WEB3,IBFT,EEA

File: docs/Privacy/Configuring-Privacy.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Privacy/Configuring-Privacy.md]: The command line itself includes all four so I'm not sure I need to re-include the others here when I'm explaining the privacy focused options.  Happy to discuss if you feels it's misleading though.

File: docs/Privacy/Configuring-Privacy.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Privacy/Configuring-Privacy.md]: Added EEA here

File: docs/Privacy/Configuring-Privacy.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Privacy/Configuring-Privacy.md]: Added EEA here

File: docs/Privacy/Configuring-Privacy.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/Privacy/Configuring-Privacy.md]: And here

File: docs/Privacy/Configuring-Privacy.md (hunk 5 above)
Comment (MadelineMurray) [reply docs/Privacy/Configuring-Privacy.md]: And here

File: docs/Privacy/Configuring-Privacy.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Privacy/Configuring-Privacy.md]: Spoke to Sally and changed the wording to make it clearer that EEA is specified alongside the other options.

**Pull request merged**